src/testmode_init.py

```diff
@@ -0,0 +1,40 @@
+import time
```

Comment:
Why it's here and not in the tests module? End user doesn't need this.

Reply:
this file will be refactored once #1794 is merged. Currently there is no nice way to put it inside the tests.